CMakeLists.txt

```diff
@@ -186,56 +146,64 @@ if (PACKAGE_MANAGER STREQUAL "conan")
         set(CMAKE_RUNTIME_OUTPUT_DIRECTORY_DEBUG ${CMAKE_RUNTIME_OUTPUT_DIRECTORY})
     endif ()
 
+    find_package(Threads REQUIRED)
+
+    find_package(Angelscript CONFIG REQUIRED)
+    find_package(Vorbis CONFIG REQUIRED)
+    find_package(OpenAL CONFIG REQUIRED)
+    find_package(SDL2 CONFIG REQUIRED)
+    find_package(sdl2-image CONFIG REQUIRED)
+    find_package(sdl2-ttf CONFIG REQUIRED)
+    find_package(LibNoise REQUIRED)
 
     list(APPEND _link_libraries
-            CONAN_PKG::sdl2
-            CONAN_PKG::sdl2_image
-            CONAN_PKG::sdl2_ttf
-            CONAN_PKG::sdl2_ttf
-            CONAN_PKG::libnoise
-            CONAN_PKG::zlib
```

Comment:
why remove zlib? is it linked somewhere else? it's used for savegame compression.
it's linked farther down when ENABLE_ANGELSCRIPT is true, but zlib will always be needed for compliation